### PR TITLE
Config loader and Readme

### PR DESCRIPTION
**Pull request by nomad-actual**

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -141,11 +145,25 @@ async function run() {
   }
 }
 
-log("Starting service");
 
-cron.schedule(config.cron, run);
+function init() {
+  console.log('init - why not use pino')
```
Comment (enigmaneo) [inline src/index.ts]: use `function log(message: string)` to log messages.

File: src/index.ts [hunk 2]
```diff
@@ -141,11 +145,25 @@ async function run() {
   }
 }
 
-log("Starting service");
 
-cron.schedule(config.cron, run);
+function init() {
```
Comment (enigmaneo) [inline src/index.ts]: Why not make this an async function?

File: src/index.ts [hunk 3]
```diff
@@ -141,11 +145,25 @@ async function run() {
...
+    .then((loadedConf: Config) => {
+      config = loadedConf;
+
+      log("Starting service");
+
+      cron.schedule(config.cron, run);
+      
+      run();
+    }).catch((e) => {
+      console.log('Error initializing...', e)
```
Comment (enigmaneo) [inline src/index.ts]: Use log method `async function log(message: string)`

**Review by enigmaneo — CHANGES_REQUESTED** [review]
Use the log file to log instead of the console.

File: src/index.ts (hunk 1 above)
Comment (enigmaneo) [inline src/index.ts]: Why add a pino where the only thing that would be useful from it is time. Use the log function.

Comment (nomad-actual) [reply src/index.ts]: Can't log before the config is loaded.

Also, I would but I was just following the patterns already present

https://github.com/rmontgomery2018/rssGetto/blob/main/src/index.ts#L58
https://github.com/rmontgomery2018/rssGetto/blob/main/src/index.ts#L132

**Review by enigmaneo — APPROVED** [review]

**Pull request merged**